main.go

```diff
@@ -30,6 +31,7 @@ ORDER BY created DESC`
 func main() {
 	var debug bool
 	p := params{}
+	flag.StringVar(&p.htmlOutput, "html-output", "", "Generate HTML report to this file (use dash [-] for stdout)")
 	flag.StringVar(&p.csvOutput, "csv-output", "", "Convert XML to a CSV file (use dash [-] for stdout)")
 	flag.StringVar(&p.jiraUrl, "jira-url", "https://issues.redhat.com/", "Url of JIRA instance")
```

Comment:
Mention (or even better check or mangle) that this needs a trailing slash?

Reply:
Fixed, I used `url` instead of `string` so no string concatenation just `url.Parse`